source/administration/bucket-replication.rst

```diff
@@ -75,7 +75,7 @@ replication while adding the following MinIO-only features:
   buckets.
 
 - Multi-Site replication of objects between three or more MinIO deployments
-  (New in :minio-release:`RELEASE.2021-09-23T04-46-24Z`).
+  (Available since :minio-release:`RELEASE.2021-09-23T04-46-24Z`).
```

Comment:
We can just drop this, it's been more than a year.

Reply:
dropped